Xamarin.Forms.Core/Markup/ElementExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Markup
+{
+	public static class ElementExtensions
+	{
+		public static TElement Effects<TElement>(this TElement element, params Effect[] effects) where TElement : Element
+		{
+			for (int i = 0; i < effects.Length; i++)
+				element.Effects.Add(effects[i]);
+			return element;
+		}
+
+		public static TFontElement FontSize<TFontElement>(this TFontElement fontElement, double size) where TFontElement : Element, IFontElement
```

Comment:
that looks like a property name, not a method name. it's confusing

Reply:
It is intended to resemble a property name, since these helpers are shortcuts to property setters.
This is part of a fluent API design strategy I used, which I recently [explained on Twitter](https://twitter.com/vincenth_net/status/1219695514276573184):
> Fluent api's have strong context, which you can leverage in naming methods. You can leave out noise words like with, set, do etc.
>
> Don't name the methods for clarity in isolation, but for clarity in their intended usage context.